shared/core/types/relay.go

```diff
@@ -0,0 +1,53 @@
+package types
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+)
+
+var (
+	errInvalidRelayInvalidPayload  = errors.New("invalid relay payload")
+	errInvalidJSONRPCInvalidRPC    = errors.New("invalid value for JSONRPC field")
```

Comment:
NIT: Do we need the second `Invalid` in the naming of these variables? I think can be omitted.

Reply:
Thank you, fixed.